app/vmui/packages/vmui/src/utils/uplot/scatter.ts

```diff
@@ -0,0 +1,105 @@
+import uPlot, { OrientCallback } from "uplot";
+
+const deg360 = 2 * Math.PI;
+
+export const drawPoints = (u: uPlot, seriesIdx: number) => {
+  const size = 4 * uPlot.pxRatio;
+  const r = size / 2;
+  const squareSize = size * 1.2;
+  const squareHalf = squareSize / 2;
```

Comment:
Can you make these numbers constants because it's not clear what these numbers mean and look like magic numbers?